code/parachain/frame/cosmwasm/src/runtimes/abstraction.rs

```diff
@@ -148,7 +148,7 @@ impl Gas {
 		if self.checkpoints.len() < 2 {
 			return Err(NoCheckpointToPop)
 		}
-		let child = self.checkpoints.pop().unwrap();
+		let child = self.checkpoints.pop().expect("account by external caller");
```

Comment:
Ditto.  This never fails because we’ve just checked that length is at least two.

Reply:
if let skip 1 pop

functional handling is awesome

i just fixed linter, i fully agree with liner. 

sure better fix is doing skip 1 pop map ok or err. code was bad. 8 do not fix code

Reply:
now. just ensured linter runs on ci. i am for fixing it in proper skip pop map or or err, but that was not point now. point was to get linter.

Reply:
i will send functional approach in next pr.

Reply:
it will be fun with mute. my point is that whole plhandling is broken. it is easy to write provable code. just issue is that there are tons of shit code to handle...